rust/tracing/Cargo.toml

```diff
@@ -25,6 +25,7 @@ uuid.workspace = true
 whoami.workspace = true
 
 [dev-dependencies]
+criterion = "0.5.1"
```

Comment:
does not seem referenced. does it do its magic just by virtue of being a dependency?

Reply:
why do we need this - or should we remove that new dependency?

Reply:
Like I said, it was not compiling on my machine, but I can remove it

Reply:
either it's a local setup issue, or we're missing something in the CI that exercises this dependency.
I would prefer not to add a dependency without knowing why exactly pulls on it.

Reply:
got it!
![image](https://github.com/user-attachments/assets/4a74c572-c9ac-4019-8f78-d3b50139c9b4)
`cargo bench`triggers it.